### PR TITLE
DOC: Minor fix to _hist_bin_fd documentation

### PR DESCRIPTION
**Pull request by nglinh**
This function returns 0 when IQR is 0, instead of 1 as mentioned in docstring

### CODE REVIEW

**Comment by seberg** [comment]
Well, this is a bit confusing, since it returns 0 for the bin-width, which is then later translated to 1 for the number of bins.

File: numpy/lib/histograms.py [hunk 1]
```diff
@@ -207,7 +207,7 @@ def _hist_bin_fd(x, range):
     than the standard deviation, so it is less accurate, especially for
     long tailed distributions.
 
-    If the IQR is 0, this function returns 1 for the number of bins.
+    If the IQR is 0, this function returns 0 for the number of bins.
```
Comment (rossbar) [inline numpy/lib/histograms.py]:
```suggestion
    If the IQR is 0, this function returns 0 for the bin width.
```
`_hist_bin_fd` returns bin widths, not number of bins - this should clear up the issue

**Review by rossbar — CHANGES_REQUESTED** [review]
It seems the main source of confusion is with the fact that the `_hist_bin_fd` estimator returns an optimal bin width rather than a number of bins. There is an explicit note in `_hist_bin_auto` explaining how the bin width estimate from `fd` is handled in special cases.

File: numpy/lib/histograms.py (hunk 1 above)
Comment (anirudh2290) [reply numpy/lib/histograms.py:210]: gentle ping, can we accept this suggested change and merge ?

Comment (seberg) [reply numpy/lib/histograms.py]: Thanks for the ping, lets just do that.

**Pull request merged**